### PR TITLE
Added pageZoomLevel option to image renderer setup documentation

### PR DESCRIPTION
**Pull request by kaffarell**
Documentation fix when https://github.com/grafana/grafana-image-renderer/pull/387 merges

### CODE REVIEW

**Comment by kaffarell** [comment]
The pr on grafana-image-renderer is still up.

**Comment by kaffarell** [comment]
this is ready to be merged!

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 1]
```diff
@@ -432,3 +432,18 @@ Limit the maximum device scale factor that can be requested. Default is `4`.
   }
 }
 ```
+
+#### Page zoom level
+
+Set a page zoom. Default is `1`, `1.5` equals 150% zoom.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:438]:
```suggestion
The following command sets a page zoom level. The default value is `1`. A value of `1.5` equals 150% zoom.
```

**Review by chri2547 — APPROVED** [review]
Suggestion to wording, approving.

**Comment by joanlopez** [comment]
> this is ready to be merged!

Can you please run `prettier` to format docs? cc/ @kaffarell 

Thanks!

**Comment by kaffarell** [comment]
Prettier run done @joanlopez !

**Review by joanlopez — APPROVED** [review]
LGTM!

**Pull request merged**